From f341c015ef39afba1b87768075c8e7e0e068778b Mon Sep 17 00:00:00 2001
From: Michael Sarahan <msarahan@gmail.com>
Date: Wed, 31 Aug 2022 08:52:53 -0500
Subject: [PATCH 1/1] comment execProcess stdout/stderr

---
 src/core/process.ts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/core/process.ts b/src/core/process.ts
index d0a2475..e743fb7 100644
--- a/src/core/process.ts
+++ b/src/core/process.ts
@@ -31,8 +31,8 @@ export async function execProcess(
     const process = Deno.run({
       ...options,
       stdin: stdin !== undefined ? "piped" : options.stdin,
-      stdout: typeof (options.stdout) === "number" ? options.stdout : "piped",
-      stderr: typeof (options.stderr) === "number" ? options.stderr : "piped",
+      //stdout: typeof (options.stdout) === "number" ? options.stdout : "piped",
+      //stderr: typeof (options.stderr) === "number" ? options.stderr : "piped",
     });
 
     if (stdin !== undefined) {
-- 
2.34.1

