From fd3024606586e0f0db5e19aa0cff6b301a5d3692 Mon Sep 17 00:00:00 2001
From: Michael Sarahan <msarahan@gmail.com>
Date: Wed, 31 Aug 2022 11:31:57 -0500
Subject: [PATCH 1/1] gate removal of css file based on existence

---
 src/core/sass.ts | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/core/sass.ts b/src/core/sass.ts
index eb2942b..c5d00ef 100644
--- a/src/core/sass.ts
+++ b/src/core/sass.ts
@@ -332,7 +332,9 @@ export async function compileWithCache(
       } catch (error) {
         // Compilation failed, so clear out the output file
         // which will be invalid CSS
-        Deno.removeSync(outputFilePath);
+        if (existsSync(outputFilePath)) {
+          Deno.removeSync(outputFilePath);
+        }
         throw error;
       }
       cacheIndex[identifierHash] = { key: cacheIdentifier, hash: inputHash };
-- 
2.34.1

