From 4d7588c5471bded3708190270bf1d17417df88eb Mon Sep 17 00:00:00 2001
From: Michael Sarahan <msarahan@gmail.com>
Date: Thu, 1 Sep 2022 13:12:45 -0500
Subject: [PATCH 1/1] truthiness for stdout/stderr execProcess

---
 src/core/process.ts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/core/process.ts b/src/core/process.ts
index e743fb7..6f9f726 100644
--- a/src/core/process.ts
+++ b/src/core/process.ts
@@ -101,7 +101,7 @@ export async function execProcess(
       closeStream(process.stderr);
     } else {
       // Process the streams independently
-      if (process.stdout !== null) {
+      if (process.stdout) {
         stdoutText = await processOutput(
           iterateReader(process.stdout),
           options.stdout,
@@ -110,7 +110,7 @@ export async function execProcess(
         process.stdout.close();
       }
 
-      if (process.stderr != null) {
+      if (process.stderr) {
         const iterator = stderrFilter
           ? filteredAsyncIterator(iterateReader(process.stderr), stderrFilter)
           : iterateReader(process.stderr);
-- 
2.34.1

