From e6abb0facb1e410508f489427eaefe2119b49642 Mon Sep 17 00:00:00 2001
From: Michael Sarahan <msarahan@gmail.com>
Date: Wed, 31 Aug 2022 10:19:27 -0500
Subject: [PATCH 1/1] use CONDA_PREFIX as user dir when set

---
 src/core/appdirs.ts   | 12 ++++++++++++
 1 files changed, 12 insertions(+)

diff --git a/src/core/appdirs.ts b/src/core/appdirs.ts
index f259a8c..c32abd3 100644
--- a/src/core/appdirs.ts
+++ b/src/core/appdirs.ts
@@ -36,6 +36,10 @@ function quartoDir(
 }
 
 export function userDataDir(appName: string, roaming = false) {
+  const condaPrefix = Deno.env.get("CONDA_PREFIX");
+  if (condaPrefix) {
+    return join(condaPrefix, "share", appName);
+  }
   switch (Deno.build.os) {
     case "darwin":
       return darwinUserDataDir(appName);
@@ -58,6 +62,10 @@ export function userConfigDir(appName: string, roaming = false) {
 }
 
 export function userCacheDir(appName: string) {
+  const condaPrefix = Deno.env.get("CONDA_PREFIX");
+  if (condaPrefix) {
+    return join(condaPrefix, "share", appName);
+  }
   switch (Deno.build.os) {
     case "darwin":
       return darwinUserCacheDir(appName);
@@ -69,6 +77,10 @@ export function userCacheDir(appName: string) {
 }
 
 export function userRuntimeDir(appName: string) {
+  const condaPrefix = Deno.env.get("CONDA_PREFIX");
+  if (condaPrefix) {
+    return join(condaPrefix, "share", appName);
+  }
   switch (Deno.build.os) {
     case "darwin":
       return darwinUserCacheDir(appName);
-- 
2.34.1

